### PR TITLE
en,zh: add v1.4 GA release notes and whats new

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

- add v1.4 GA release notes
- add what's new in v1.4

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/whats-new-in-v1.4.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: TiDB Operator v1.4 新特性
+---
+
+# TiDB Operator v1.4 新特性
+
+TiDB Operator v1.4 引入了以下关键特性，从扩展性、易用性等方面帮助你更轻松地管理 TiDB 集群及其周边工具。
+
+## 兼容性改动
```
Comment (ran-huang) [inline zh/whats-new-in-v1.4.md:9]: 这里要包括从 v1.3 到 v1.4 所有的兼容性变更吗？

File: zh/releases/release-1.4.0.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+发布日期: 2022 年 12 月 29 日
+
+TiDB Operator 版本：1.4.0
+
+## 新功能
+
+- 支持使用新的 `TidbDashboard` CRD 独立管理 [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) ([#4787](https://github.com/pingcap/tidb-operator/pull/4787), [@SabaPing](https://github.com/SabaPing))
+- 支持为 TiKV 与 PD 配置 Liveness Probe ([#4763](https://github.com/pingcap/tidb-operator/pull/4763), [@mikechengwei](https://github.com/mikechengwei))
+
```
Comment (fengou1) [inline zh/releases/release-1.4.0.md:15]:
```suggestion
支持基于 Amazon EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698)，[@gozssky] (https://github.com/gozssky)
```

Comment (fengou1) [reply zh/releases/release-1.4.0.md:15]: 新功能里面加一句话，目的让用户看得更加清晰。

File: en/releases/release-1.4.0.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: TiDB Operator 1.4.0 Release Notes
+---
+
+# TiDB Operator 1.4.0 Release Notes
+
+Release date: Dec 29, 2022
```
Comment (TomShawn) [inline en/releases/release-1.4.0.md:7]:
```suggestion
Release date: December 29, 2022
```

File: en/releases/release-1.4.0.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+TiDB Operator version: 1.4.0
+
+## New features
+
+- Support managing [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) in a separate `TidbDashboard` CRD ([#4787](https://github.com/pingcap/tidb-operator/pull/4787), [@SabaPing](https://github.com/SabaPing))
+- Support configuring Liveness Probe for TiKV and PD ([#4763](https://github.com/pingcap/tidb-operator/pull/4763), [@mikechengwei](https://github.com/mikechengwei))
+
+## Improvements
+
+- Adapt to the new TiProxy version ([#4802](https://github.com/pingcap/tidb-operator/pull/4802), [@xhebox](https://github.com/xhebox))
```
Comment (TomShawn) [inline en/releases/release-1.4.0.md:18]: What is the new version?

File: zh/releases/release-1.4.0.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+## 优化提升
+
+- 适配新的 TiProxy 版本 ([#4802](https://github.com/pingcap/tidb-operator/pull/4802), [@xhebox](https://github.com/xhebox))
```
Comment (TomShawn) [inline zh/releases/release-1.4.0.md:18]: 新版本是什么版本呢？

File: en/tidb-operator-overview.md [hunk 6]
```diff
@@ -13,6 +13,7 @@ The corresponding relationship between TiDB Operator and TiDB versions is as fol
 | TiDB versions | Compatible TiDB Operator versions |
 |:---|:---|
 | dev               | dev                 |
+| TiDB >= 6.5       | 1.4 (Recommended)，1.3 |
```
Comment (TomShawn) [inline en/tidb-operator-overview.md:16]:
```suggestion
| TiDB >= 6.5       | 1.4 (Recommended), 1.3 |
```

File: en/whats-new-in-v1.4.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Due to the change in [#4493](https://github.com/pingcap/tidb-operator/pull/4494), if you deploy TiCDC without setting the `log-file` configuration item, the rolling upgrade of TiDB Operator to v1.4.0-alpha.1 and later versions will cause TiCDC to be rolling recreated.
+
+## Extensibility
+
+- Support managing [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) in a separate `TidbDashboard` CRD.
+- Support backup and restore based on Amazon EBS volume-snapshot.
+- Support scaling in or out multiple TiKV and TiFlash Pods at the same time.
+- Support restoring a cluster to a specific time point from a snapshot backup or a log backup using BR.
+- Support modifying the IOPS and throughput of AWS EBS storage used by a TiDB cluster.
+- Support TiProxy. (Experimental feature)
```
Comment (TomShawn) [inline en/whats-new-in-v1.4.md:25]:
```suggestion
- Support TiProxy as an experimental feature.
```

**Review by TomShawn — APPROVED** [review]
only minor comments

File: zh/releases/release-1.4.0.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+发布日期: 2022 年 12 月 29 日
+
+TiDB Operator 版本：1.4.0
+
+## 新功能
+
+- 支持使用新的 `TidbDashboard` CRD 独立管理 [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) ([#4787](https://github.com/pingcap/tidb-operator/pull/4787), [@SabaPing](https://github.com/SabaPing))
+- 支持为 TiKV 与 PD 配置 Liveness Probe ([#4763](https://github.com/pingcap/tidb-operator/pull/4763), [@mikechengwei](https://github.com/mikechengwei))
+支持基于 Amazon EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698)，[@gozssky] (https://github.com/gozssky)
```
Comment (ran-huang) [inline zh/releases/release-1.4.0.md:15]:
```suggestion
- 支持基于 Amazon EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698)，[@gozssky] (https://github.com/gozssky)
```

File: en/releases/release-1.4.0.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+Release date: December 29, 2022
+
+TiDB Operator version: 1.4.0
+
+## New features
+
+- Support managing [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) in a separate `TidbDashboard` CRD ([#4787](https://github.com/pingcap/tidb-operator/pull/4787), [@SabaPing](https://github.com/SabaPing))
+- Support configuring Liveness Probe for TiKV and PD ([#4763](https://github.com/pingcap/tidb-operator/pull/4763), [@mikechengwei](https://github.com/mikechengwei))
+
```
Comment (ran-huang) [inline en/releases/release-1.4.0.md:15]:
```suggestion
- Support backup and restore based on Amazon EBS volume-snapshot ([#4698](https://github.com/pingcap/tidb-operator/pull/4698)，[@gozssky] (https://github.com/gozssky)
```

File: zh/releases/release-1.4.0.md (hunk 5 above)
Comment (ran-huang) [reply zh/releases/release-1.4.0.md:19]: @csuzhangxc

Comment (csuzhangxc) [reply zh/releases/release-1.4.0.md:19]: 现在 TiProxy 还没有明确的版本号，要不这条移除算了吧

File: en/whats-new-in-v1.4.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+aliases: ['/tidb-in-kubernetes/dev/whats-new-in-v1.3']
+---
+
+# What's New in TiDB Operator 1.4
+
+TiDB Operator 1.4 introduces the following key features, which helps you manage TiDB clusters and the tools more easily in terms of extensibility and usability.
+
+## Compatibility changes
+
+Due to the change in [#4683](https://github.com/pingcap/tidb-operator/pull/4683), the feature of modifying storage is disabled by default. If you want to scale out the PVC for a component, you need to first enable this feature.
```
Comment (csuzhangxc) [inline en/whats-new-in-v1.4.md:12]: 英文要不用 `resize` 替换 `scale out`? ref https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/

File: zh/whats-new-in-v1.4.md (hunk 1 above)
Comment (csuzhangxc) [reply zh/whats-new-in-v1.4.md:10]: 应该就是只有这一条

File: en/whats-new-in-v1.4.md (hunk 10 above)
Comment (ran-huang) [inline en/whats-new-in-v1.4.md:12]:
```suggestion
Due to the change in [#4683](https://github.com/pingcap/tidb-operator/pull/4683), the feature of modifying storage is disabled by default. If you want to resize the PVC for a component, you need to first enable this feature.
```

**Review by fengou1 — APPROVED** [review]

File: en/releases/release-1.4.0.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+## New features
+
+- Support managing [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) in a separate `TidbDashboard` CRD ([#4787](https://github.com/pingcap/tidb-operator/pull/4787), [@SabaPing](https://github.com/SabaPing))
+- Support configuring Liveness Probe for TiKV and PD ([#4763](https://github.com/pingcap/tidb-operator/pull/4763), [@mikechengwei](https://github.com/mikechengwei))
+- Support backup and restore based on Amazon EBS volume-snapshot ([#4698](https://github.com/pingcap/tidb-operator/pull/4698)，[@gozssky](https://github.com/gozssky)
+
+## Improvements
+
+- Support configuring `.spce.preferIPv6: true` for compatibility with IPv6 network environments ([#4811](https://github.com/pingcap/tidb-operator/pull/4811)，[@KanShiori](https://github.com/KanShiori)
```
Comment (ran-huang) [inline en/releases/release-1.4.0.md:19]:
```suggestion
- Support configuring `.spec.preferIPv6: true` for compatibility with IPv6 network environments ([#4811](https://github.com/pingcap/tidb-operator/pull/4811)，[@KanShiori](https://github.com/KanShiori)
```

File: zh/releases/release-1.4.0.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+## 新功能
+
+- 支持使用新的 `TidbDashboard` CRD 独立管理 [TiDB Dashboard](https://github.com/pingcap/tidb-dashboard) ([#4787](https://github.com/pingcap/tidb-operator/pull/4787), [@SabaPing](https://github.com/SabaPing))
+- 支持为 TiKV 与 PD 配置 Liveness Probe ([#4763](https://github.com/pingcap/tidb-operator/pull/4763), [@mikechengwei](https://github.com/mikechengwei))
+- 支持基于 Amazon EBS 的 TiDB 集群 volume-snapshot 的备份和恢复 ([#4698](https://github.com/pingcap/tidb-operator/pull/4698)，[@gozssky](https://github.com/gozssky)
+
+## 优化提升
+
+- 支持配置 `.spce.preferIPv6: true` 兼容 IPv6 网络环境 ([#4811](https://github.com/pingcap/tidb-operator/pull/4811)，[@KanShiori](https://github.com/KanShiori)
```
Comment (ran-huang) [inline zh/releases/release-1.4.0.md:19]:
```suggestion
- 支持配置 `.spec.preferIPv6: true` 兼容 IPv6 网络环境 ([#4811](https://github.com/pingcap/tidb-operator/pull/4811)，[@KanShiori](https://github.com/KanShiori)
```

**Review by KanShiori — APPROVED** [review]

**Pull request merged**